mon-pix/app/metrics-adapters/plausible-adapter.js

```diff
@@ -0,0 +1,48 @@
+import BaseAdapter from 'ember-metrics/metrics-adapters/base';
+
+export default class PlausibleAdapter extends BaseAdapter {
+  static ADAPTER_ID = PlausibleAdapter.name;
+  static SCRIPT_ID = 'pix-matomo-tag-manager-analytics';
+
+  toStringExtension() {
+    return PlausibleAdapter.ADAPTER_ID;
+  }
+
+  install() {
+    const { siteId, scriptUrl } = this.config;
+
+    window.plausible =
+      window.plausible ||
+      function (...args) {
+        (window.plausible.q = window.plausible.q || []).push(args);
+      };
+
+    const scriptElement = document.createElement('script');
+    const firstScriptElement = document.getElementsByTagName('script')[0];
+    scriptElement.type = 'text/javascript';
+    scriptElement.defer = true;
+    scriptElement.async = true;
+    scriptElement.src = scriptUrl;
+    scriptElement.setAttribute('data-domain', siteId);
+    scriptElement.id = PlausibleAdapter.SCRIPT_ID;
+    firstScriptElement.parentNode.insertBefore(scriptElement, firstScriptElement);
+  }
+
+  identify() {}
+
+  trackEvent({ 'pix-event-name': eventName, ...rest }) {
+    window.plausible(eventName, { props: rest });
+  }
```

Comment:
Uniformisation avec l'autre methode 

```suggestion
  trackEvent({ 'pix-event-name': eventName, ...props }) {
    window.plausible(eventName, props: { ...props });
  }
```

Reply:
done